packages/api/src/schema/schema.yaml

```diff
@@ -3289,3 +3289,14 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/error"
+
+  /room:
+    post:
+      description: create a new room
+      responses:
```

Comment:
Since you're adding it here, mind documenting the success respnose schema? can just do an inline schema here with an `id` field etc